streams/src/test/java/org/apache/kafka/test/NoOpProcessorContext.java

```diff
@@ -120,6 +120,16 @@ public Cancellable schedule(final Duration interval,
     @Override
     public void commit() {}
 
+    @Override
+    public long currentSystemTimeMs() {
+        throw new UnsupportedOperationException("this method is not supported in NoOpProcessorContext");
```

Comment:
Nit: I guess we could add support if needed. Maybe better say `"Not implemented yet."` ? (Same below.)